docs/new-dependency-policy.md

```diff
@@ -0,0 +1,31 @@
+# New Package Dependency Policy
+
+This is the policy for adding a new package dependency to the Cardano SL suite of package.
+
+1. Whenever a new dependency is added to an existing Cardano SL package that fact should be
+   noted in the Github Pull Request (PR).
+
+2. If the new dependency is already a direct or indirect dependency of an existing
+   Cardano SL package that should be noted in the Github PR. Existing dependencies do
+   not need further veting.
+
+3. If the new dependency is not already a direct or indirect dependency of an existing
+   Cardano SL package the following information should be noted in the PR:
+
+   * The Hackage entry for the new package.
+   * What functionality the new package provides for Cardano SL.
+   * The level of risk the new package provides.
```

Comment:
I would add:

* Are there competing packages that provide this functionality? If so, can you list a brief justification for selecting this package over the others?